### PR TITLE
Add note related to 'never,task' rule in auditd

### PR DESCRIPTION
**Pull request by jotacarma90**
## Description
In this PR we are adding a note in the advanced settings of FIM to warn that in some systems auditd comes by default with a rule that prevents Wazuh (and any other process) of the audit rules from working properly.

### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 1]
```diff
@@ -67,6 +67,33 @@ You need to install the audit daemon if you don’t have it already installed on
          # cp /usr/sbin/audisp-af_unix /sbin/audisp-af_unix
          # rc-service auditd restart
 
+.. note::
+
+   On certain systems, auditd is configured with a default rule that causes all audit rule processing to be skipped. This disables any whodata-related configuration of Wazuh.
+   
+   Link to auditd Linux manual: https://man7.org/linux/man-pages/man8/auditctl.8.html#top_of_page
```
Comment (MarcelKemp) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:74]:
```suggestion
   Link to auditd Linux manual: https://man7.org/linux/man-pages/man8/auditctl.8.html#DISABLED_BY_DEFAULT
```

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 2]
```diff
@@ -67,6 +67,33 @@ You need to install the audit daemon if you don’t have it already installed on
...
+
+   Check in the auditd rules:
```
Comment (MarcelKemp) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:76]:
```suggestion
   Check auditd rules:
```

**Review by MarcelKemp — CHANGES_REQUESTED** [review]

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst (hunk 1 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/file-integrity/advanced-settings.rst]: done

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst (hunk 2 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/file-integrity/advanced-settings.rst]: done

**Review by MarcelKemp — APPROVED** [review]

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 3]
```diff
@@ -67,6 +67,33 @@ You need to install the audit daemon if you don’t have it already installed on
...
+   On certain systems, auditd is configured with a default rule that causes all audit rule processing to be skipped. This disables any whodata-related configuration of Wazuh.
+   
+   Link to auditd Linux manual: https://man7.org/linux/man-pages/man8/auditctl.8.html#DISABLED_BY_DEFAULT
+
+   Check auditd rules:
+  
+   .. code-block::
+
+      auditctl -l
+      -a never,task
```
Comment (javimed) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:81]: What is `-a never,task` here? Is this the output of the `auditctl -l` command? Is this the real output or an output when piping the grep command? We could use a separate output code-block to show the output. Also, commands in code-blocks need to include the prompt symbol. `code-block` directives need to specify a language for syntax highlighting, for example for Linux command lines we use `.. code-block:: console`.

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 4]
```diff
@@ -67,6 +67,33 @@ You need to install the audit daemon if you don’t have it already installed on
...
+
+   You need to remove that rule, or include a specific filter for the syscheck daemon in the auditd rule file ``/etc/audit/rules.d/audit.rules``:
```
Comment (javimed) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:83]: What's our recommendation? Removing or filtering?

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 5]
```diff
@@ -67,6 +67,33 @@ You need to install the audit daemon if you don’t have it already installed on
...
+  
+   .. code-block::
+
+      -a always,task -F exe=‘/var/ossec/bin/wazuh-syscheckd’
+      -a never,task
```
Comment (javimed) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:88]: if we are keeping this rule we need to emphasize the filter line with for example `:emphasize-lines: 1` in this code block

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 6]
```diff
@@ -67,6 +67,33 @@ You need to install the audit daemon if you don’t have it already installed on
          # cp /usr/sbin/audisp-af_unix /sbin/audisp-af_unix
          # rc-service auditd restart
 
+.. note::
```
Comment (javimed) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:70]: Why a note? Why not just inserting all this in the enabling who-data monitoring steps that follow below?

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 7]
```diff
@@ -67,6 +67,33 @@ You need to install the audit daemon if you don’t have it already installed on
...
+
+   On certain systems, auditd is configured with a default rule that causes all audit rule processing to be skipped. This disables any whodata-related configuration of Wazuh.
```
Comment (javimed) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:72]: Certain systems such as which ones?

**Review by javimed — CHANGES_REQUESTED** [review]

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst (hunk 3 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/file-integrity/advanced-settings.rst:81]: Done

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst (hunk 4 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/file-integrity/advanced-settings.rst:83]: Filtering, done

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst (hunk 5 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/file-integrity/advanced-settings.rst:88]: Done

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst (hunk 7 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/file-integrity/advanced-settings.rst:72]: Done

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst (hunk 6 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/file-integrity/advanced-settings.rst]: Paragraph modified with a new layout outside the note!

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 8]
```diff
@@ -67,6 +67,39 @@ You need to install the audit daemon if you don’t have it already installed on
...
          # rc-service auditd restart
 
+Check possible DISABLED BY DEFAULT status of auditd
+
+#. On most systems auditd is configured with a default rule (``-a never,task``) that causes all audit rule processing to be skipped. This disables any whodata-related configuration of Wazuh.
+ 
+   Link to auditd Linux manual: https://man7.org/linux/man-pages/man8/auditctl.8.html#DISABLED_BY_DEFAULT
+ 
+   Check auditd rules:
+
```
Comment (javimed) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:77]:
```suggestion
In most systems, auditd includes a rule to skip processing of every audit rule by default. This setting prevents the reporting of any whodata-related information. To ensure that auditd is not `DISABLED BY DEFAULT <https://man7.org/linux/man-pages/man8/auditctl.8.html#DISABLED_BY_DEFAULT>`__, follow these steps.

#. Check the output of this command to find out if the auditd rules include the ``-a never,task`` rule.

```

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 9]
```diff
@@ -67,6 +67,39 @@ You need to install the audit daemon if you don’t have it already installed on
...
+#. On most systems auditd is configured with a default rule (``-a never,task``) that causes all audit rule processing to be skipped. This disables any whodata-related configuration of Wazuh.
+ 
+   Link to auditd Linux manual: https://man7.org/linux/man-pages/man8/auditctl.8.html#DISABLED_BY_DEFAULT
+ 
+   Check auditd rules:
+ 
+   .. code-block:: console
+ 
+     auditctl -l
```
Comment (javimed) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:80]:
```suggestion
      # auditctl -l | grep task
```

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 10]
```diff
@@ -67,6 +67,39 @@ You need to install the audit daemon if you don’t have it already installed on
...
+ 
+#. If the output shows the above mentioned rule:
+ 
+   .. code-block:: console
+ 
+     -a never,task
+ 
+   You need to include a specific filter for the syscheck daemon, over the other rule, in the auditd rule file ``/etc/audit/rules.d/audit.rules``:
```
Comment (javimed) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:88]:
```suggestion
#. If the output displays the ``-a never,task`` rule, add the following filter rule in ``/etc/audit/rules.d/audit.rules``. Make sure to place it before the mentioned rule.
```

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 11]
```diff
@@ -67,6 +67,39 @@ You need to install the audit daemon if you don’t have it already installed on
...
+ 
+   .. code-block::
+     :emphasize-lines: 1
+ 
+     -a always,task -F exe=‘/var/ossec/bin/wazuh-syscheckd’
+     -a never,task
```
Comment (javimed) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:94]:
```suggestion
   .. code-block:: none
      :emphasize-lines: 1
 
      -a always,task -F exe=‘/var/ossec/bin/wazuh-syscheckd’
      -a never,task
```

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 12]
```diff
@@ -67,6 +67,39 @@ You need to install the audit daemon if you don’t have it already installed on
...
+ 
+#. After that, restart auditd and Wazuh agent to apply the changes:
+ 
+   .. code-block:: console
+ 
+     systemctl restart auditd
+     systemctl restart wazuh-agent
```
Comment (javimed) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:101]: Use three-spaces tabulation

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 13]
```diff
@@ -67,6 +67,39 @@ You need to install the audit daemon if you don’t have it already installed on
...
+ 
 Perform the following steps to enable who-data monitoring. In this example, you configure who-data monitoring for ``/etc`` directory.
```
Comment (javimed) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:103]:
```suggestion
Next, perform the following steps to enable who-data monitoring. In this example, you configure who-data monitoring for the ``/etc/`` directory.
```

**Review by javimed — CHANGES_REQUESTED** [review]

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst (hunk 8 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/file-integrity/advanced-settings.rst:77]: done

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst (hunk 9 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/file-integrity/advanced-settings.rst:80]: done

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst (hunk 10 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/file-integrity/advanced-settings.rst:88]: done

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst (hunk 11 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/file-integrity/advanced-settings.rst:94]: done

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst (hunk 12 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/file-integrity/advanced-settings.rst:101]: done

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst (hunk 13 above)
Comment (jotacarma90) [reply source/user-manual/capabilities/file-integrity/advanced-settings.rst:103]: done

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 14]
```diff
@@ -67,7 +67,34 @@ You need to install the audit daemon if you don’t have it already installed on
...
 
-Perform the following steps to enable who-data monitoring. In this example, you configure who-data monitoring for ``/etc`` directory.
+In most systems, auditd includes a rule to skip processing of every audit rule by default. This setting prevents the reporting of any whodata-related information. To ensure that auditd is not `DISABLED BY DEFAULT <https://man7.org/linux/man-pages/man8/auditctl.8.html#DISABLED_BY_DEFAULT>`__, follow these steps.
+
+#. Check the output of this command to find out if the auditd rules include the ``-a never,task`` rule.
+
+   Link to auditd Linux manual: https://man7.org/linux/man-pages/man8/auditctl.8.html#DISABLED_BY_DEFAULT
+ 
+   Check auditd rules:
+
```
Comment (javimed) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:77]:
```suggestion
```

**Review by javimed — CHANGES_REQUESTED** [review]

**Review by javimed — APPROVED** [review]

**Review by MarcelKemp — APPROVED** [review]

**Pull request merged**